### PR TITLE
feat(apache_postgresql): add website monitoring template for Apache and Postgesql sites

### PR DESCRIPTION
**Pull request by mhall119**
Add website monitoring template for Apache/Posgresql deployments

Signed-off-by: Michael Hall <mhall119@gmail.com>

### CODE REVIEW

File: apache_postgresql/website_template.yml [hunk 1]
```diff
@@ -0,0 +1,569 @@
...
+kind: Bucket
+metadata:
+    name: bettering_faraday_5b4005
+spec:
+    associations:
+      - kind: Label
+        name: modest_wu_5b4001
+    name: website
+    retentionRules:
+      - everySeconds: 2.592e+06
```
Comment (jsteenb2) [inline apache_postgresql/website_template.yml:19]: what does this translate too?

Comment (mhall119) [reply apache_postgresql/website_template.yml:19]: 30 days

Comment (jsteenb2) [reply apache_postgresql/website_template.yml:19]: not crazy about our `everySeconds` rule... thinking out loud here. Would be cool to allow something like the following if you ask me. WDYT?

```
- every: 30
  period: day
```

Comment (mhall119) [reply apache_postgresql/website_template.yml:19]: I personally think that the actual content of the template file should be largely ignored by users and developers alike, the interface with them should be at the tools or UI level, so I'm not especially worried about how we serialize retention periods in the file.

Comment (jsteenb2) [reply apache_postgresql/website_template.yml:19]: that's fair. I speaking more to the API of influxdb here than the actual pkg itself. What you see here is a reflection of the current API. I only just happened to dawn on me this API was like that when I saw your 30 day in seconds number 😬.

Comment (mhall119) [reply apache_postgresql/website_template.yml:19]: if you're going to change it, could you use the same syntax as Flux? So something like:
```
- every: 30d
- every: 24h
```

Comment (jsteenb2) [reply apache_postgresql/website_template.yml:19]: yah, that'd be even better 👍

**Pull request merged**